tf-psa-crypto/include/psa/crypto_sizes.h

```diff
@@ -146,8 +146,10 @@
 #define PSA_HASH_MAX_SIZE 32u
 #elif defined(PSA_WANT_ALG_SHA_224) || defined(PSA_WANT_ALG_SHA3_224)
 #define PSA_HASH_MAX_SIZE 28u
-#else /* SHA-1 or smaller */
+#elif defined(PSA_WANT_ALG_SHA_1)
 #define PSA_HASH_MAX_SIZE 20u
+#else /* None of the above defined */
```

Comment:
This is not correct for a configuration with only MD4 or RIPEMD160. Such a configuration is unlikely, but not impossible.

Reply:
Sorry, I meant MD5, not MD4.